runtime/Makefile

```diff
@@ -13,7 +13,7 @@ DN_OBJS=$(addprefix third-party/decNumber/, decQuad.o decContext.o)
 DN_DEFINES=-DHAVE_STDBOOL_H -DHAVE_STDINT_H -DDECPRINT=0
 DTOA_OBJS=third-party/dtoa/emyg_dtoa.o
 THIRD_PARTY_OBJS=$(DTOA_OBJS) $(BT_OBJS) $(DN_OBJS)
-RT_OBJS=alloc.o error.o list.o main.o mapping.o structure.o panic.o print.o string.o int.o decimal.o
+RT_OBJS=alloc.o error.o list.o main.o mapping.o structure.o panic.o print.o string.o int.o decimal.o tagged_to_tagged.o
```

Comment:
file name should be something like `convert.c`

Reply:
Changed